### PR TITLE
Feature/configurable accent color

### PR DESCRIPTION
**Pull request by jMaddrell**
I'll work on moving UIConfig to json in a subsequent PR.

I've also updated the PIO & NPM dependencies, if this is merged we can close all of the dependabot generated PRs

### CODE REVIEW

**Comment by jMaddrell** [comment]
Didn't intend on adding the smaller gifs to this branch but it does seem to fix the reallocation failures for me can revert if needed

**Comment by jMaddrell** [comment]
Whoops wrong levering icon will fix

File: Firmware/data/AP_Config.gif
Comment (DiverOfDark) [inline Firmware/data/AP_Config.gif:1]: Looks like animation is lost?

File: Firmware/src/scenes/Styles.h [hunk 1]
```diff
@@ -1,33 +1,38 @@
...
-    if (!redText48initialized) {
-      lv_style_init(&redText48);
-      lv_style_set_text_font(&redText48, &font_48);
-      lv_style_set_text_color(&redText48, lv_color_hex(0xFF0000));
-      redText48initialized = true;
+  lv_style_t *getAccentText48() {
+    if (!accentText48initialized) {
+      lv_style_init(&accentText48);
+      lv_style_set_text_font(&accentText48, &font_48);
+      lv_style_set_text_color(&accentText48, lv_color_hex(getAccentColor()));
```
Comment (DiverOfDark) [inline Firmware/src/scenes/Styles.h:33]: NB for future PR - after changing settings we would need to update style.

File: Firmware/data/AP_Config.gif
Comment (jMaddrell) [reply Firmware/data/AP_Config.gif:1]: I’d removed other languages from this image, I can restore them and decompress if you’d like?

File: Firmware/src/scenes/Styles.h (hunk 1 above)
Comment (jMaddrell) [reply Firmware/src/scenes/Styles.h:33]: I was tempted to trigger a reboot to apply the settings change but will also look into invalidating the styles.

File: Firmware/data/AP_Config.gif
Comment (DiverOfDark) [reply Firmware/data/AP_Config.gif:1]: Ok, no problem if it was intentional. Although I like the feel of original image - 16 kb of saved space seems more beneficial.

Comment (jMaddrell) [reply Firmware/data/AP_Config.gif:1]: I also like the feel of the original I can re-add if you'd like? Adding the other frames brings the image to 17K vs 1.2K

**Pull request merged**